collections/Channels.js

```diff
@@ -62,3 +62,13 @@ Channel.getLatest = function(limit) {
     limit: limit
   });
 }
+
+/*
+ * Find a single channel by id
+ *
+ * @param {string} id The channel id to fetch
+ * @return {Mongo.Cursor} the resulting Mongo cursor from find()
+ */
+Channel.getChannel = function(id) {
+  return Channel.find({"_id": id});
+}
```

Comment:
It turns out you can just do `Channel.findOne("someIDstring")` so this method is kind of unnecessary. 

The `find` and `findOne` methods are equivalent to the Meteor collection calls. Check out the [findOne doc entry](http://docs.meteor.com/#/full/findone).